compiler/rustc_mir_transform/src/inline.rs

```diff
@@ -440,6 +440,10 @@ impl<'tcx> Inliner<'tcx> {
             validation: Ok(()),
         };
 
+        for var_debug_info in callee_body.var_debug_info.iter() {
```

Comment:
What's this for?

Reply:
(answered in: https://github.com/rust-lang/rust/pull/113802#issuecomment-1640709427 )